### ISSUE TITLE
Cannot reproduce the performance of the trained model 

### ISSUE THREAD

**Issue by acforvs** [ZiyuanMa/DHC#7]
Hi, thank you for this repo!

I used the hyperparameters from `configs.py` for training, but after 150k+ steps, the model's performance has nearly changed (and is still bad), and the loss seems to be nearly the same as well (~ 0.0042)

I was wondering if you have made any changes to the parameters and if you would be able to share them 

Thanks!

**Comment by wanggsz** [comment]
Hi, I am trying to run the code yet meet a problem. I'm wondering if you would be able to help me with it. Thanks a lot!
I was hoping to re-train the model, and I used `python3 train.py` however it showed that there's a typeerror regarding the package ray. I start to question if I gave the wrong command and how to train a DHC model?
It would be of great help if you can show me how you start training this model. Thanks!!

**Comment by acforvs** [comment]
Hi @wanggsz, I slightly refactored the code, you can find my extended version here https://github.com/acforvs/dhc-robust-mapf. It also contains some scripts to measure the performance on MovingAI maps.

There were several issues with the config file in the original repo that made it impossible to replicate the performance.

**Comment by wanggsz** [comment]
Hi, @acforvs 
Thank you for the reply. I’ll check out the link. Thanks a lot! :)
